lib/galaxy/tools/data/__init__.py

```diff
@@ -362,6 +366,11 @@ def configure_and_load(self, config_element, tool_data_path, from_shed_config=Fa
                     log.debug('Loading Data Table URL "%s" as filename "%s".', filename, tmp_file.name)
                     filename = tmp_file.name
                     tmp_file.flush()
+                else:
+                    # Pull the filename from a global config
+                    filename = file_element.get('from_config', None) or None
+                    if filename:
+                        filename = self.other_config_dict.get(filename, None)
```

Comment:
```suggestion
                        filename = self.other_config_dict.get(filename)
```
`None` is returned by default

Reply:
explicit is better than implicit